### PR TITLE
fix #72128 bbc.com

### LINKED ISSUE #72128: bbc.com
### Issue URL (Ads)
 [https://www.bbc.com/news](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.bbc.com%2Fnews)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Safari v        1.8.10
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media
Other extensions used: | none

### PR DESCRIPTION
**Pull request by stanislav-atr**
#72128

### CODE REVIEW

**Comment by Alex-302** [comment]
I don't see any element.
@AdguardTeam/filters-maintainers please take a look.

**Comment by zloyden** [comment]
I can see only last requests with `dotcom`, but not the elements.
So, I partly agree with those changes.

File: AnnoyancesFilter/sections/cookies_specific.txt
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:117]: this element is already blocked by 
`bbc.com##header[role="banner"] div[dir="ltr"][class*="-Wrapper "]:not([class*="-Banner"])`

**Comment by Sergey-Lyapin** [comment]
and I don't see the rest of elements and requests

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
> and I don't see the rest of elements and requests

I see.

**Pull request merged**
